.changes/remove-on-page-load.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri": patch:breaking
+---
+
+Removed the `on_page_load` hook from `Plugin` and `Builder`. Use `on_navigation` instead.
```

Comment:
But `on_navigation` is different from `on_page_load` and we have `on_page_load` in wry now

Reply:
we do??? totally missed that
the on_page_load we have right now is yiky

Reply:
The new implementation is using native hooks